src/Microsoft.Health.Dicom.Api/Extensions/HttpResponseExtensions.cs

```diff
@@ -12,12 +13,27 @@ namespace Microsoft.Health.Dicom.Api.Extensions
 {
     internal static class HttpResponseExtensions
     {
+        public const string ErroneousAttributesHeader = "x-ms-erroneous-attributes";
```

Comment:
Let's not use `x-ms`. This convention has been deprecated. Perhaps just `erroneous-attributes` or `erroneous-dicom-attributes`

Reply:
As the header conversation scope is 'global', I feel we may want to indicate `dicom`. will use `erroneous-dicom-attributes` .